lib/lambda_ethereum_consensus/beacon_chain/state_transition.ex

```diff
@@ -0,0 +1,56 @@
+defmodule BeaconChain.StateTransition do
```

Comment:
should we create another module called process block or block processing? Otherwise this one will become gigantic

Reply:
Yes let's do that.